CHANGES.txt

```diff
@@ -1,4 +1,27 @@
-0.4.1
+0.5.0
```

Comment:
Probably should include the date. Otherwise, this pull request looks good to me.